src/components/ui/icons/Set.vue

```diff
@@ -43,12 +48,12 @@
 
 <script>
   import Widget from '../../vuestic-components/vuestic-widget/VuesticWidget'
-  import Slider from 'vue-slider-component/src/vue2-slider.vue'
+  import VuesticSlider from 'components/vuestic-components/vuestic-slider/VuesticSlider'
```

Comment:
Add slider to vuestic-components-plugin.js and remove useless import

Reply:
Ah, right!
Thanks for pointing this out.